1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -0,0 +1,15 @@
+
+export function fizzbuzz(input: number): string {
+    switch (true) {
+        case input % 3 === 0 && input % 5 === 0:
+            return "fizzbuzz";
+        case input < 0:
+            return "error negative number";
```

Comment:
according the greading checklist the range is between 0 & 100, so you could check for edge case at the beginning of the code :
```ts
   if (input < 0 || input > 100) {
        return "error: input out of range, should be between 1 & 100";
    }
```